From 952ef891f04bd201a0a08840d5ed1cc534cf249e Mon Sep 17 00:00:00 2001
From: "Gogineni, GiribabuX" <giribabux.gogineni@intel.com>
Date: Fri, 27 Oct 2017 19:30:32 +0530
Subject: [PATCH 01/31] ASoC: Intel: Skylake: Add ULL machine driver entry

This patch adds ULL machine driver name to driver
data.

Change-Id: Ib6e47b14f7e7c8a24a3bdad4034a39008de7f3be
Signed-off-by: Gogineni, GiribabuX <giribabux.gogineni@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/19014
Reviewed-by: Tewani, Pradeep D <pradeep.d.tewani@intel.com>
Reviewed-by: Shaik, Kareem M <kareem.m.shaik@intel.com>
Reviewed-by: Koul, Vinod <vinod.koul@intel.com>
Reviewed-by: Kale, Sanyog R <sanyog.r.kale@intel.com>
Reviewed-by: audio_build <audio_build@intel.com>
Tested-by: Sm, Bhadur A <bhadur.a.sm@intel.com>
---
 sound/soc/intel/skylake/skl.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 5fefb53..e2f15e8 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -1072,6 +1072,11 @@ static void skl_remove(struct pci_dev *pci)
 		.drv_name = "bxt_tdf8532",
 		.fw_filename = "intel/dsp_fw_bxtn.bin",
 	},
+	{
+		.id = "INT34C3",
+		.drv_name = "bxt_ivi_ull",
+		.fw_filename = "intel/dsp_fw_ull_bxtn.bin",
+	},
 	{}
 };
 
-- 
1.9.1

